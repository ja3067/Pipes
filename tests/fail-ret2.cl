def foo() -> str:
	x = [1, 2, 3]
	if False:
		x = "hello"
	return x

print(foo())
