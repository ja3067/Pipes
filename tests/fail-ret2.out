RuntimeError: invalid return type (expected str)
